docs/src/utils/reader.ts

```diff
@@ -15,11 +15,18 @@ export async function getNavigationMap() {
       const { discriminant, value } = link;
       const page = discriminant === 'page' && value ? pagesBySlug[value] : null;
       const url = discriminant === 'url' ? value : `/docs/${page?.slug}`;
+      const comingSoon = discriminant === 'coming-soon' ? true : false;
+      const comingSoonBadgeLabel =
```

Comment:
Is there a case where coming soon is `true` but there is no `comingSoonBadgeLabel`?
If not we could probably just have the `comingSoonBadgeLabel` variable to pass down as props everywhere, and use that as the test case.

Reply:
> Is there a case where coming soon is true but there is no comingSoonBadgeLabel?

Probably not 😅 That was more me playing with "how would I do that if I wanted to" so totally happy to do whichever

Thinking about the use case we might not even need to customise the badge label, we could just decide that it should always be "Soon" (is that good btw?).. Could really just pass down a comingSoon boolean if that option was selected.. What you reckon? @BarnabyBishop

Reply:
> Thinking about the use case we might not even need to customise the badge label
Since this is temporary, and only for one link, you are right we probably don't need the custom label.

Reply:
Have removed that conditional field now and simplified 👍🏻 🚚  @BarnabyBishop